ecc/bls12381/ff/gen.go

```diff
@@ -0,0 +1,92 @@
+// +build ignore
+
+// Code Generation using fiat-crypto
+//
+// Download and unpack `ExtractionOCaml-Ubuntu LTS` from
+// https://github.com/mit-plv/fiat-crypto/suites/3241842351/artifacts/75305272
```

Comment:
Youch. This sounds like a fun time on the mac. Well, that's ok: we'll manage.

Reply:
The artifacts also contain the .ml files from which the binaries are built, so you could rebuild them from source.  I can also get the Mac and Windows CIs to start uploading artifacts again; I seem to recall disabling them because it seemed that seemed plausibly related to the Mac CIs failing.